pkg/daemon/server/service/rater/rater.go

```diff
@@ -217,13 +221,11 @@ func sleep(ctx context.Context, duration time.Duration) {
 // getPodReadCounts returns the total number of messages read by the pod
 // since a pod can read from multiple partitions, we will return a map of partition to read count.
 func (r *Rater) getPodReadCounts(vertexName, podName string) *PodReadCount {
-	readTotalMetricName := "forwarder_data_read_total"
```

Comment:
LGTM. nit: I intentionally put this var inside this particular func instead of const, to make it unaccessible from other funcs. A small abstraction.

Reply:
moved it back.